README.md

```diff
@@ -183,6 +183,80 @@ By making a contribution to this project, I certify that:
 
 (c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
 
+## Stability Policy
+
+The most important consideration in every code change is the impact it will have, positive or negative, on the ecosystem (modules and applications). To this end, all Collaborators must work collectively to ensure that changes do not needlessly break backwards compatibility, introduce performance or functional regressions, or negatively impact the usability of the Project on any of the platforms officially targeted for support by the Project. The TSC is responsible for determining the list of supported platforms.
+
+When discussing stability, it is important to first review Node.js' layered architecture:
+
+```
+   +---------------------------------------------------+
+   |          Module and Application Ecosystem         |
+   +---------------------------------------------------+
+        |                |    |           |          |
+        |                | +----------------------+  |
+        |                | |  Native Abstractions |  |
+        |                | |      for Node.js     |  |
+        |                | +----------------------+  |
+        |                |     |             |       |
+   +----------------+    |     |             |       /
+   |  Node.js Core  |    |     |             |      /
+   |  Library API   |    |     |             |     /
+   +----------------+    |     |             |    /
+   |     js impl    |    |     |             |   /
+   +----------------+    |     |             |  /
+            |            |     |             | /
+   +--------------------------------------+  |/
+   | Node.js Application Binary Interface |  |
+   +--------------------------------------|  |
+   |              C/C++ impl              |  |
+   +--------------------------------------+  |
+                      |                      |
+        +---------------------------------------+
+        | Dependencies: v8, libuv, openssl, etc |
+        +---------------------------------------+
+```
```

Comment:
Nan should be used twice as much? Lol... Thanks will fix
On Apr 7, 2015 2:06 PM, "Jeremiah Senkpiel" <notifications@github.com>
wrote:

> In README.md
> <https://github.com/jasnell/dev-policy/pull/21#discussion_r27920435>:
>
> > +   |  Node.js Core  |    |     |             |      /
> > +   |  Library API   |    |     |             |     /
> > +   +----------------+    |     |             |    /
> > +   |     js impl    |    |     |             |   /
> > +   +----------------+    |     |             |  /
> > +            |            |     |             | /
> > +   +--------------------------------------+  |/
> > +   | Node.js Application Binary Interface |  |
> > +   +--------------------------------------|  |
> > +   |              C/C++ impl              |  |
> > +   +--------------------------------------+  |
> > +                      |                      |
> > +        +---------------------------------------+
> > +        | Dependencies: v8, libuv, openssl, etc |
> > +        +---------------------------------------+
> > +```
>
> Just a nit, but, should only be one line above NAN :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jasnell/dev-policy/pull/21/files#r27920435>.
>